geopyspark/geotrellis/layer.py

```diff
@@ -255,12 +255,63 @@ def to_tiled_layer(self, extent=None, layout=None, crs=None, tile_size=256,
             ``extent`` and ``layout`` must both be defined if they are to be used.
 
         Returns:
-            :class:`~geopyspark.geotrellis.rdd.TiledRasterLayer`
+            :class:`~geopyspark.geotrellis.rdd.RasterLayer`
         """
 
         return self.tile_to_layout(self.collect_metadata(extent, layout, crs, tile_size),
                                    resample_method)
 
+    def bands(self, band):
+        """Select a subsection of bands from the ``Tile``\s within the layer.
+
+        Note:
+            There could be potential high performance cost if operations are performed
+            between two sub-bands of a large data set.
+
+        Note:
+            Due to the natue of GeoPySpark's backend, if selecting a band that is out of bounds
+            then the error returned will be a ``py4j.protocol.Py4JJavaError`` and not a normal
+            Python error.
```

Comment:
I was never to get it to work. There is currently an open issue for it https://github.com/bartdag/py4j/issues/153